### PR TITLE
Docs-md --> Docs

### PR DESCRIPTION
**Pull request by Quantalabs**
The new `docs-md` branch has updated docstrings in markdown, and uses pdoc, generating more readable documentation, as can be viewed [here](https://epispot.github.io/epispot/epispot/).

### CODE REVIEW

**Comment by quantum9Innovation** [comment]
Get rid of TravisCI on #docs after the merge

**Comment by quantum9Innovation** [comment]
Also increase code snippet readability after merge

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install flake8
+
```
Comment (quantum9Innovation) [inline .github/workflows/build.yml:23]: Remove whitespace

File: .github/workflows/coverage.yml [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
-      run: |
-        python -m coverage run -m pytest
-    - name: Generate Coverage Report
-      continue-on-error: true
+    - name: Run Tests
       run: |
-        python -m coverage xml
-    - name: Upload to codecov
+        coverage run -m pytest
+    - name: Upload to Codecov
```
Comment (quantum9Innovation) [inline .github/workflows/coverage.yml:31]: Why is code coverage being changed on a docs branch?

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        pip install pdoc3
+        pip install -r requirements.txt
+    - name: Generate documentation
+      run: |
+        pdoc --html --output-dir docs epispot --template-dir pdoc3_templates --force
```
Comment (quantum9Innovation) [inline .github/workflows/docs.yml:22]: - [ ] Where is this being saved to?
 - [ ] Does this override current documentation?
- [ ] Has this workflow been tested?

File: sitemap.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (quantum9Innovation) [inline sitemap.xml:1]: This sitemap will need to be edited for the new URL

**Review by quantum9Innovation — CHANGES_REQUESTED** [review]

File: .github/workflows/docs.yml (hunk 3 above)
Comment (Quantalabs) [reply .github/workflows/docs.yml:22]: Saved to docs folder. Everything works and has been tested. This overrides any existing find to get up to date docs.

File: .github/workflows/coverage.yml (hunk 2 above)
Comment (Quantalabs) [reply .github/workflows/coverage.yml:31]: This came from changes in the master branch.

**Comment by quantum9Innovation** [comment]
This PR is closed
**Do not merge a master-based branch into a nightly-based branch**

**Pull request closed without merging**